src/test/java/seedu/guest/model/util/SampleDataUtilTest.java

```diff
@@ -54,22 +55,22 @@ public void getSampleGuestBook() {
         GuestBook exceptedGb = new GuestBook();
         exceptedGb.addGuest(new Guest(new Name("James Oliver"), new Phone("98345344"),
                 new Email("jamesoliver@example.com"), new DateRange("15/09/22 - 19/09/22"),
-                new NumberOfGuests("2"), new IsRoomClean("no"), new Bill()));
+                new NumberOfGuests("2"), new IsRoomClean("no"), new Bill(), new Request()));
         exceptedGb.addGuest(new Guest(new Name("Kayla Smith"), new Phone("91823471"),
                 new Email("kaylasmith@example.com"), new DateRange("22/01/23 - 25/01/23"),
-                new NumberOfGuests("3"), new IsRoomClean("no"), new Bill("10")));
+                new NumberOfGuests("3"), new IsRoomClean("no"), new Bill("10"), new Request("hi")));
```

Comment:
Should we have more appropriate examples for passing in the string value to construct a Request?